### PR TITLE
feat(qna3e): new docs

### PR DESCRIPTION
**Pull request by gc87**

### CODE REVIEW

**Review by fengzeroz — APPROVED** [review]

File: zh_CN/south-devices/mitsubishi-3e/fx5u.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Neuron 设置
+
+1. 在 Neuron 南向设备管理中添加一个 Mitsubishi 3E 设备。
+
+2. 在设备配置中修改 `PLC IP 地址` 为目标设备 IP 地址。
+
+3. 在设备配置中修改 `PLC 端口` 为目标设备端口，提交设置表单。
+![fx5u6](./assets/fx5u6.jpg)
+
+6. 添加 `组`，添加测试 `点位`。
```
Comment (lixiumei123) [inline zh_CN/south-devices/mitsubishi-3e/fx5u.md:31]: change 6 to 4.

File: zh_CN/south-devices/mitsubishi-3e/q03ude.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Neuron 设置
+
+1. 在 Neuron 南向设备管理中添加一个 Mitsubishi 3E 设备。
+
+2. 在设备配置中修改 `PLC IP 地址` 为目标设备 IP 地址。
+
+3. 在设备配置中修改 `PLC 端口` 为目标设备端口，提交设置表单。
+![q03ude5](./assets/q03ude5.jpg)
+
+6. 添加 `组`，添加测试 `点位`。
```
Comment (lixiumei123) [inline zh_CN/south-devices/mitsubishi-3e/q03ude.md:28]: change 6 to 4.

File: en_US/south-devices/mitsubishi-3e/fx5u.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+## Neuron Setting
+
+1. Add a Mitsubishi 3E device to Neuron Southbound Device Management.
+
+2. Change the `PLC IP Address` to the target device IP address in the device configuration.
+
+3. Modify `PLC Port` as the target device port in the device configuration and submit the setup form.
+![fx5u6](./assets/fx5u_en6.jpg)
+
+6. Add `Group`，Add test `tag`.
```
Comment (lixiumei123) [inline en_US/south-devices/mitsubishi-3e/fx5u.md:31]: change 6 to 4.

File: en_US/south-devices/mitsubishi-3e/q03ude.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+## Neuron Setting
+
+1. Add a Mitsubishi 3E device to Neuron Southbound Device Management.
+
+2. Change the `PLC IP Address` to the target device IP address in the device configuration.
+
+3. Modify `PLC Port` as the target device port in the device configuration and submit the setup form.
+![q03ude5](./assets/q03ude_en5.jpg)
+
+6. Add `Group`，Add test `tag`.
```
Comment (lixiumei123) [inline en_US/south-devices/mitsubishi-3e/q03ude.md:28]: change 6 to 4.

File: en_US/south-devices/mitsubishi-3e/fx5u.md (hunk 3 above)
Comment (gc87) [reply en_US/south-devices/mitsubishi-3e/fx5u.md]: fixed

File: zh_CN/south-devices/mitsubishi-3e/q03ude.md (hunk 2 above)
Comment (gc87) [reply zh_CN/south-devices/mitsubishi-3e/q03ude.md]: fixed

File: en_US/south-devices/mitsubishi-3e/q03ude.md (hunk 4 above)
Comment (gc87) [reply en_US/south-devices/mitsubishi-3e/q03ude.md]: fixed

**Comment by joey8869** [comment]
There is a conflict in the directory file.

**Pull request closed without merging**